### PR TITLE
added linux commands to the cheatsheet

### PR DESCRIPTION
**Pull request by SaumitraPatil**
### Checklist:
- [x] Read our [contributing guidelines](../docs/CONTRIBUTING.md).
- [x] Search for duplicates.
- [x] Used an informative name for this pull request.

### Follow-up
- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: CheatSheets/linux-cheatsheet.md [hunk 1]
```diff
@@ -29,8 +30,9 @@
 | `ls -al` | Formatted list of contents for the directory including hidden files |
 | `rm file` | Delete file | 
 | `rm -r dir` | Delete directory *dir* | 
-| `rm -f file` |Force delete file | 
-| `rm -rf dir` | Force delete directory *dir* | 
+| `rm -f file` |Force deletes file | 
+| `rm -rf dir` | Force deletes the directory *dir* |
```
Comment (crescentpartha) [inline CheatSheets/linux-cheatsheet.md:34]:
```suggestion
| `rm -rf dir` | Forcibly delete the directory "dir" | 
```

File: CheatSheets/linux-cheatsheet.md [hunk 2]
```diff
@@ -29,8 +30,9 @@
 | `ls -al` | Formatted list of contents for the directory including hidden files |
 | `rm file` | Delete file | 
 | `rm -r dir` | Delete directory *dir* | 
-| `rm -f file` |Force delete file | 
-| `rm -rf dir` | Force delete directory *dir* | 
+| `rm -f file` |Force deletes file | 
+| `rm -rf dir` | Force deletes the directory *dir* | 
+| `rm -rfv fir` | Force deletes the directory *dir* with verbose output |
```
Comment (crescentpartha) [inline CheatSheets/linux-cheatsheet.md:35]:
```suggestion
| `rm -rfv dir` | Forcibly delete the directory "dir" with verbose output |
```

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Everything is Ok, just fix the typo mistake.

**Review by SaumitraPatil — COMMENTED** [review]
changes done :+1:

**Review by crescentpartha — APPROVED** [review]
Approved your changes.

**Pull request merged**